pcstac/Dockerfile

```diff
@@ -14,4 +14,7 @@ RUN pip install -e ./pccommon -e ./pcstac[server]
 ENV APP_HOST=0.0.0.0
 ENV APP_PORT=81
 
+#TEMP
+ENV USE_API_HYDRATE=true
```

Comment:
TODO: Flagging to remove this before merging. Using now to publish image for use without var set from the helm chart.

Reply:
Removed.